File: Multiples of 3 or 5(PE)/main.cpp
#include <iostream>
int main()
{
    int n, i = 1, sum = 0;
    std::cout << "Enter N:";
    std::cin >> n;
    while (i <= n)
    {
        if (i % 3 == 0 || i % 5 == 0)
        {
            sum += i;
        }
        i++;
    }
    std::cout << "Sum: " << sum << "\n";
    return 0;
}

File: Multiples of 3 or 5(PE)/main.js
function main() {
    let i = 1, sum = 0;
    while (i <= 1000) {
        if (i % 3 == 0 || i % 5 == 0) {
            sum += i;
        }
        i++;
    }
    console.log("Sum: ", sum)
}
main()